doc/api/fs.md

```diff
@@ -469,12 +469,55 @@ no effect on Windows (will behave like `fs.constants.F_OK`).
 
 The final argument, `callback`, is a callback function that is invoked with
 a possible error argument. If any of the accessibility checks fail, the error
-argument will be populated. The following example checks if the file
-`/etc/passwd` can be read and written by the current process.
+argument will be populated. 
 
 ```js
-fs.access('/etc/passwd', fs.constants.R_OK | fs.constants.W_OK, (err) => {
-  console.log(err ? 'no access!' : 'can read/write');
+// Check if `package.json` exists in the current directory.
+fs.access('./package.json', fs.constants.F_OK, (err) => {
+  console.log(err ? 'No package.json file!' : 'package.json exists!');
+});
+```
+
+```js
+// Check if we can read `package.json` file.
```

Comment:
Please avoid using informal pronouns like "we" in the docs

Reply:
...so maybe `Check if package.json file can be read` or something similar to that...

Reply:
Thanks @jasnell and @Trott, I'm learning a lot of things. I'll keep these things in my mind.